sim/core/attack.go

```diff
@@ -366,11 +366,10 @@ func (unit *Unit) EnableAutoAttacks(agent Agent, options AutoAttackOptions) {
 
 		DamageMultiplier: 1,
 		ThreatMultiplier: 1,
+		BonusCoefficient: TernaryFloat64(options.MainHand.GetSpellSchool() == SpellSchoolPhysical, 1, 0),
```

Comment:
Not for this PR, and pre-existing: SpellSchool should always be Physical, and by extension, BonusCoefficient always be 1 here.

The SpellConfig can be modified from the outside, anyhow, so Shadowfiend (which might just as well have BonusCoefficient 1, guessing from "Observed 6 times stronger AP scaling then expected") and even more so possible enemy mobs could just customize outside of default options.